frontend/src/components/object/ObjectFileDetails.vue

```diff
@@ -81,68 +97,113 @@ onBeforeMount( async () => {
+          </span>
+          <span v-else>
+            File details
+          </span>
         </h1>
       </div>
+      <div class="flex col justify-content-start">
+        <div class="flex col align-items-center heading">
+          <font-awesome-icon
+            icon="fa-solid fa-circle-info"
+            style="font-size: 2rem"
+          />
+          <h1 class="pl-1 font-bold">
+            {{ filename }}
+          </h1>
+        </div>
+
+        <div
+          class="action-buttons"
+        >
+          <ShareObjectButton
+            :id="props.objectId"
+          />
+          <DownloadObjectButton
+            v-if="obj.public || permissionStore.isObjectActionAllowed(
+              props.objectId, getUserId, Permissions.READ, bucketId)"
+            :mode="ButtonMode.ICON"
+            :ids="[props.objectId]"
+          />
+          <Button
+            v-if="permissionStore.isObjectActionAllowed(
+              props.objectId, getUserId, Permissions.MANAGE, bucketId)"
+            class="p-button-lg p-button-text"
+            @click="showPermissions(props.objectId)"
+          >
+            <font-awesome-icon icon="fa-solid fa-users" />
+          </Button>
+          <DeleteObjectButton
+            v-if="permissionStore.isObjectActionAllowed(
+              props.objectId, getUserId, Permissions.DELETE, bucketId)"
+            :mode="ButtonMode.ICON"
+            :ids="[props.objectId]"
+            @on-deleted-success="onDeletedSuccess"
+          />
+        </div>
+      </div>
+    </div>
 
-      <div
-        class="action-buttons"
-      >
-        <ShareObjectButton
-          :id="props.objectId"
+
+    <div class="flex flex-row">
+      <div class="flex flex-column w-6 gap-3 py-5">
+        <ObjectProperties
+          :object-id="props.objectId"
+          :version-id="props.versionId"
+          :full-view="true"
         />
-        <DownloadObjectButton
-          v-if="obj.public || permissionStore.isObjectActionAllowed(
-            props.objectId, getUserId, Permissions.READ, bucketId)"
-          :mode="ButtonMode.ICON"
-          :ids="[props.objectId]"
+        <ObjectAccess :object-id="props.objectId" />
+        <ObjectMetadata
+          :object-id="props.objectId"
+          :version-id="props.versionId"
         />
-        <Button
-          v-if="permissionStore.isObjectActionAllowed(
-            props.objectId, getUserId, Permissions.MANAGE, bucketId)"
-          class="p-button-lg p-button-text"
-          @click="showPermissions(props.objectId)"
-        >
-          <font-awesome-icon icon="fa-solid fa-users" />
-        </Button>
-        <DeleteObjectButton
-          v-if="permissionStore.isObjectActionAllowed(
-            props.objectId, getUserId, Permissions.DELETE, bucketId)"
-          :mode="ButtonMode.ICON"
-          :ids="[props.objectId]"
-          @on-deleted-success="onDeletedSuccess"
+      </div>
+      <div>
+        <Divider layout="vertical" />
+      </div>
```

Comment:
Does this have to be in its own div wrapper or can it standalone without the wrapper?